web/src/components/apps/AppVersionHistoryRow.jsx

```diff
@@ -246,47 +301,101 @@ class AppVersionHistoryRow extends Component {
     return (
       <div className="flex flex1 justifyContent--flexEnd alignItems--center">
         {this.renderReleaseNotes(version)}
-  
+
         <div>
-          {version.status === "pending_preflight" ?
+          {version.status === "pending_preflight" ? (
             <div className="u-marginRight--10 u-position--relative">
               <Loader size="30" />
-              <p className="checks-running-text u-fontSize--small u-lineHeight--normal u-fontWeight--medium">Running checks</p>
+              <p className="checks-running-text u-fontSize--small u-lineHeight--normal u-fontWeight--medium">
+                Running checks
+              </p>
             </div>
-          :
-          <div>
-            <Link to={`/app/${app?.slug}/downstreams/${app?.downstream.cluster?.slug}/version-history/preflight/${version?.sequence}`}
-              className="icon preflightChecks--icon u-marginRight--10 u-cursor--pointer u-position--relative"
-              data-tip="View preflight checks">
-              {preflightState.preflightsFailed || preflightState.preflightState === "warn" || newPreflightResults ?
-                <div>
-                  <span className={`icon version-row-preflight-status-icon ${preflightState.preflightsFailed ? "preflight-checks-failed-icon" : preflightState.preflightState === "warn" ? "preflight-checks-warn-icon" : ""}`} />
-                  <p className={`checks-running-text u-fontSize--small u-lineHeight--normal u-fontWeight--medium ${preflightState.preflightsFailed ? "err" : preflightState.preflightState === "warn" ? "warning" : newPreflightResults ? "success" : ""}`}>{checksStatusText}</p>
-                </div>
-                : null}
-            </Link>
-            <ReactTooltip effect="solid" className="replicated-tooltip" />
-          </div>
-          }
+          ) : preflightState.preflightState !== "" ? (
+            <div>
```

Comment:
I see a lot of opportunities in this file to use Fragments, however I realize that may be unnecessary bloat for this PR. Feel free to ignore my comments on this.

Reply:
@alicenstar good catch! I think it might be too much for this PR to update with fragments. I will put in on my radar for next time.